packages/commercetools/helpers/src/index.ts

```diff
@@ -1 +1,15 @@
-console.log('Hello commercetools helpers')
\ No newline at end of file
+import { UiMediaGalleryItem, UiProductName, UiProductPrice } from '@vue-storefront/interfaces'
+import { ProductVariant, Image, ProductData } from './types/GraphQL'
+
+export const getName = (product: ProductData): UiProductName => product.name
+
+export const getSlug = (product: ProductData): UiProductName => product.slug
```

Comment:
How about this `Agnostic` instead of UI eg. `AgnosticProductName` ? 
Also for simple strings/numbers maybe we should keep it as string/number and add those types only for objects to not confuse people? String is string. I don't see a value in renaming it 
@patzick